Mile.Project.Windows/Mile.Project.Cpp.props

```diff
@@ -140,7 +140,6 @@
       <PreprocessorDefinitions>_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <Optimization>Disabled</Optimization>
       <RuntimeLibrary Condition="'$(MileProjectEnableVCLTLSupport)' == 'true'">MultiThreadedDebug</RuntimeLibrary>
-      <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
```

Comment:
@MouriNaruto 

This setting is needed in Debug mode because CFG is not compatible with the default DebugInformationFormat setting of '/ZI'.